docs/gen-ai/mcp.md

```diff
@@ -0,0 +1,1118 @@
+| `notifications/roots/list_changed` | Notification indicating that the list of roots has changed. | ![Development](https://img.shields.io/badge/-development-blue) |
+| `notifications/tools/list_changed` | Notification indicating that the list of tools has changed. | ![Development](https://img.shields.io/badge/-development-blue) |
+| `ping` | Request to check that the other party is still alive. | ![Development](https://img.shields.io/badge/-development-blue) |
+| `prompts/get` | Request to get a prompt. | ![Development](https://img.shields.io/badge/-development-blue) |
+| `prompts/list` | Request to list prompts available on server. | ![Development](https://img.shields.io/badge/-development-blue) |
+| `resources/list` | Request to list resources available on server. | ![Development](https://img.shields.io/badge/-development-blue) |
+| `resources/read` | Request to read a resource. | ![Development](https://img.shields.io/badge/-development-blue) |
+| `resources/subscribe` | Request to subscribe to a resource. | ![Development](https://img.shields.io/badge/-development-blue) |
+| `resources/templates/list` | Request to list resource templates available on server. | ![Development](https://img.shields.io/badge/-development-blue) |
+| `resources/unsubscribe` | Request to unsubscribe from resource updates. | ![Development](https://img.shields.io/badge/-development-blue) |
+| `roots/list` | Request to list roots available on server. | ![Development](https://img.shields.io/badge/-development-blue) |
+| `sampling/createMessage` | Request to create a sampling message. | ![Development](https://img.shields.io/badge/-development-blue) |
+| `tools/call` | Request to call a tool. | ![Development](https://img.shields.io/badge/-development-blue) |
+| `tools/list` | Request to list tools available on server. | ![Development](https://img.shields.io/badge/-development-blue) |
+
+---
+
+`network.transport` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
+
+| Value | Description | Stability |
+| --- | --- | --- |
+| `pipe` | Named or anonymous pipe. | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
+| `quic` | QUIC | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
+| `tcp` | TCP | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
+| `udp` | UDP | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
+| `unix` | Unix domain socket | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
+
+<!-- prettier-ignore-end -->
+<!-- END AUTOGENERATED TEXT -->
+<!-- endsemconv -->
+
+## Metrics
+
+### Metric: `mcp.client.operation.duration`
+
+This metric is [recommended][MetricRecommended].
+
+This metric SHOULD be specified with
+[`ExplicitBucketBoundaries`](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.50.0/specification/metrics/api.md#instrument-advisory-parameters)
```

Comment:
Wondering if there was a reason that these explicit bucket boundaries were chosen as a recommendation?  The alternatives include [Exponential Histogram](https://opentelemetry.io/docs/specs/otel/metrics/data-model/#exponentialhistogram) or leaving it up to the implementor to decide which kind of histogram to use.  Especially with GenAI applications, it might not be so unusual to have > 5 minute response times.